CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1605,21 +1594,15 @@ bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta, float l
 
     //// Single pixel cluster probabilities
 
-    chi2yavgone_ = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2yavgone +
-                   yratio * thePixelTemp_[index_id_].enty[ihigh].chi2yavgone;
-    chi2yminone_ = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2yminone +
-                   yratio * thePixelTemp_[index_id_].enty[ihigh].chi2yminone;
-    chi2xavgone = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2xavgone +
-                  yratio * thePixelTemp_[index_id_].enty[ihigh].chi2xavgone;
-    chi2xminone = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2xminone +
-                  yratio * thePixelTemp_[index_id_].enty[ihigh].chi2xminone;
-
-    fracyone_ = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].fracyone +
-                yratio * thePixelTemp_[index_id_].enty[ihigh].fracyone;
-    fracytwo_ = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].fracytwo +
-                yratio * thePixelTemp_[index_id_].enty[ihigh].fracytwo;
+    chi2yavgone_ = (1.f - yratio) * enty0_->chi2yavgone + yratio * enty1_->chi2yavgone;
+    chi2yminone_ = (1.f - yratio) * enty0_->chi2yminone + yratio * enty1_->chi2yminone;
+    chi2xavgone = (1.f - yratio) * enty0_->chi2xavgone + yratio * enty1_->chi2xavgone;
+    chi2xminone = (1.f - yratio) * enty0_->chi2xminone + yratio * enty1_->chi2xminone;
+
+    fracyone_ = (1.f - yratio) * enty0_->fracyone + yratio * enty1_->fracyone;
+    fracytwo_ = (1.f - yratio) * enty0_->fracytwo + yratio * enty1_->fracytwo;
     //       for(i=0; i<10; ++i) {
-    //		    pyspare[i]=(1.f - yratio)*thePixelTemp_[index_id_].enty[ilow].yspare[i] + yratio*thePixelTemp_[index_id_].enty[ihigh].yspare[i];
+    //		    pyspare[i]=(1.f - yratio)*enty0_->yspare[i] + yratio*enty1_->yspare[i];
```

Comment:
Is this commented out code needed: add one line explaining its usage, if so; otherwise just remove

Reply:
Added comment